src/main/java/Pool.java

```diff
@@ -0,0 +1,19 @@
+import vi.viredis.client.ViRedis;
+
+public interface Pool<T> {
+
+    /*
+     * @return one of the pooled objects.
+     */
+    ViRedis get();
```

Comment:
This is a very specific type, nothing to do with T which is specified as a type parameter in the interface. Reconsider.

Reply:
done!